templates/docs/gatsby/src/pages/api-documentation.tsx

```diff
@@ -70,8 +71,14 @@ const ApiDocumentation: React.FC = () => {
   }, [openrpcQueryData]);
 
   useEffect(() => {
-    if (openrpcDocument && openrpcDocument.servers && openrpcDocument.servers[0]) {
+    if (!openrpcDocument) {
+      return;
+    }
+    if (openrpcDocument.servers && openrpcDocument.servers[0]) {
       setInspectorUrl(openrpcDocument.servers[0].url);
+      if (openrpcDocument.servers[0]["x-transport"]) {
+        setInspectorTransport(openrpcDocument.servers[0]["x-transport"]);
```

Comment:
maybe we should make a generic-ish component for 'server picker' ?

Reply:
yea we definitely should. Dunno if it should block this PR though.

Reply:
created #484 to track it